.github/workflows/dependabot_automerge.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - name: Dependabot metadata
         id: metadata
-        uses: dependabot/fetch-metadata@v2.0
+        uses: dependabot/fetch-metadata@v2
```

Comment:
Version 2.0 doesn't exist, I think aim was to not specify the patch version to reduce the version bump PRs so I opted for `v2` instead of `v2.0.0`